src/perf/lib/ThroughputClient.cpp

```diff
@@ -614,7 +614,7 @@ ThroughputClient::OnTcpConnectionComplete(
     if (Connection) {
         Connection->Close();
     }
-    CxPlatEventSet(*StopEvent);
+    //CxPlatEventSet(*StopEvent);
```

Comment:
Why was this commented out?

Reply:
> Do not merge until I remove the tcp change that forces it to always fail.

Testing that it actually works.

Reply:
Suggestion: Make it a Draft PR if you don't want it merged.